realsense2_camera/test/live_camera/test_camera_service_call.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2024 Intel Corporation. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os, sys
+import pytest
+
+sys.path.append(os.path.abspath(os.path.dirname(__file__)+"/../utils"))
+import pytest_rs_utils
+import pytest_live_camera_utils
+from pytest_rs_utils import launch_descr_with_parameters
+from pytest_rs_utils import get_node_heirarchy
+
+test_params_test_srv_d455 = {
+    'camera_name': 'D455',
+    'device_type': 'D455',
+    }
+test_params_test_srv_d435i = {
+    'camera_name': 'D435i',
+    'device_type': 'D435i',
+    }
+test_params_test_srv_d415 = {
+    'camera_name': 'D415',
+    'device_type': 'D415',
+    }
+
+'''
+The test is to check service call device_info with type DeviceInfo
+'''
+@pytest.mark.parametrize("launch_descr_with_parameters", [    
+    pytest.param(test_params_test_srv_d455, marks=pytest.mark.d455),
+    pytest.param(test_params_test_srv_d435i, marks=pytest.mark.d435i),
+    pytest.param(test_params_test_srv_d415, marks=pytest.mark.d415),    
+    ],indirect=True)
+@pytest.mark.launch(fixture=launch_descr_with_parameters)
+class TestCamera_ServiceCall(pytest_rs_utils.RsTestBaseClass):
```

Comment:
this test only uses DeviceInfo service call. So, shall we rename this function and file names such that it is specific to DeviceInfo service.

Reply:
Yes, thought of it already - There are 8 service calls available. Plan is to have rest seven of them be added here in this file, so no need of changing the file name as of now. And, about changing the function name, yes/maybe -- to be addressed when implementing rest of the service calls.

Reply:
got it, each test will carry distinct name. Function name modified. Is that fine?